jq/script-06.sh

```diff
@@ -6,3 +6,4 @@ set -euo pipefail
 # TODO: Write a command to output just the names of each player along with the score from their first attempt.
 # Your output should contain 6 lines, each with one word and one number on it.
 # The first line should be "Ahmed 1" with no quotes.
+jq -r '.[] | .name +" " + (.scores[0]| tostring)' scores.json
```

Comment:
This works, and there's nothing wrong with it, but FYI you can also do string interpolation in `jq`, so your RHS of the pipe here could read: `"\(.name) \(.scores[0])"`, and this could apply to several of your answers - I'm not saying either is better, but it's good to know :)